### ISSUE TITLE
'ms-dotnettools.dotnet-interactive-vscode' wants API proposa l 'languageConfigurationAutoClosingPairs' but that proposal DOES NOT EXIST

### ISSUE THREAD

**Issue by Kenan7** [dotnet/interactive#3331]
seeing this error

```
 WARN Via 'product.json#extensionEnabledApiProposals' extension 'ms-dotnettools.dotnet-interactive-vscode' wants API proposal 'languageConfigurationAutoClosingPairs' but that proposal DOES NOT EXIST. Likely, the proposal has been finalized (check 'vscode.d.ts') or was abandoned.
 ```

**Comment by alejandrolopezperez** [comment]
yes, what to do with this? I am trying to download the java extensions for vs code and I can't

**Comment by jonsequitur** [comment]
We haven't been able to reproduce this. Could you please provide more details including:

* What version of VS Code are you using?
* What version of the Polyglot Notebooks extension is installed?
* A screen shot.
* Any other repro steps.

**Comment by alejandrolopezperez** [comment]
thank you for getting back to me.

It happend to me when I try to download the Extension Pack for Java:

What version of VS Code are you using? *Version: 1.84.2 (user setup)*

What version of the Polyglot Notebooks extension is installed?
* v1.0.4562010*
A screen shot: [image: image.png]

Any other repro steps.just looking for it on the extensions marketplace and
it doesn't work.


Thank you

**Comment by alejandrolopezperez** [comment]
this is the whole log:

2023-11-28 18:37:30.772 [warning] Via
'product.json#extensionEnabledApiProposals' extension
'ms-dotnettools.dotnet-interactive-vscode' wants API proposal
'languageConfigurationAutoClosingPairs' but that proposal DOES NOT EXIST.
Likely, the proposal has been finalized (check 'vscode.d.ts') or was
abandoned.
2023-11-28 18:37:35.122 [info] [perf] Render performance baseline is 69ms
2023-11-28 18:37:58.844 [warning] Settings pattern "application.*" doesn't
match any settings
2023-11-28 18:47:40.511 [warning] Settings pattern "application.*" doesn't
match any settings
2023-11-28 18:52:49.193 [error] Error: Cannot read the extension from
/c:/Users/consu/.vscode/extensions/visualstudioexptteam.
vscodeintellicode-1.2.30,Cannot read the extension from
/c:/Users/consu/.vscode/extensions/visualstudioexptteam.
intellicode-api-usage-examples-0.2.8,Cannot read the extension from
/c:/Users/consu/.vscode/extensions/redhat.java-1.24.0-win32-x64,Cannot read
the extension from /c:/Users/consu/.vscode/extensions/vscjava.
vscode-java-debug-0.55.0,Cannot read the extension from
/c:/Users/consu/.vscode/extensions/vscjava.vscode-maven-0.43.0,Cannot read
the extension from /c:/Users/consu/.vscode/extensions/vscjava.
vscode-java-dependency-0.23.2
    at c:\Users\consu\AppData\Local\Programs\Microsoft VS
Code\resources\app\out\vs\code\node\sharedProcess\sharedProcessMain.js:99:78806
    at Array.reduce (<anonymous>)
    at E (c:\Users\consu\AppData\Local\Programs\Microsoft VS
Code\resources\app\out\vs\code\node\sharedProcess\sharedProcessMain.js:99:78792)
    at K.M (c:\Users\consu\AppData\Local\Programs\Microsoft VS
Code\resources\app\out\vs\code\node\sharedProcess\sharedProcessMain.js:99:70467)
    at async K.J (c:\Users\consu\AppData\Local\Programs\Microsoft VS
Code\resources\app\out\vs\code\node\sharedProcess\sharedProcessMain.js:99:68153)
    at async c:\Users\consu\AppData\Local\Programs\Microsoft VS
Code\resources\app\out\vs\code\node\sharedProcess\sharedProcessMain.js:99:64821
    at async Promise.allSettled (index 0)
    at async K.I (c:\Users\consu\AppData\Local\Programs\Microsoft VS
Code\resources\app\out\vs\code\node\sharedProcess\sharedProcessMain.js:99:64768)
    at async K.installGalleryExtensions
(c:\Users\consu\AppData\Local\Programs\Microsoft VS
Code\resources\app\out\vs\code\node\sharedProcess\sharedProcessMain.js:99:63217)
    at async K.installFromGallery
(c:\Users\consu\AppData\Local\Programs\Microsoft VS
Code\resources\app\out\vs\code\node\sharedProcess\sharedProcessMain.js:99:62538)
2023-11-28 18:52:49.227 [error] Cannot read the extension from
/c:/Users/consu/.vscode/extensions/visualstudioexptteam.
vscodeintellicode-1.2.30,Cannot read the extension from
/c:/Users/consu/.vscode/extensions/visualstudioexptteam.
intellicode-api-usage-examples-0.2.8,Cannot read the extension from
/c:/Users/consu/.vscode/extensions/redhat.java-1.24.0-win32-x64,Cannot read
the extension from /c:/Users/consu/.vscode/extensions/vscjava.
vscode-java-debug-0.55.0,Cannot read the extension from
/c:/Users/consu/.vscode/extensions/vscjava.vscode-maven-0.43.0,Cannot read
the extension from /c:/Users/consu/.vscode/extensions/vscjava.
vscode-java-dependency-0.23.2: Error: Cannot read the extension from
/c:/Users/consu/.vscode/extensions/visualstudioexptteam.
vscodeintellicode-1.2.30,Cannot read the extension from
/c:/Users/consu/.vscode/extensions/visualstudioexptteam.
intellicode-api-usage-examples-0.2.8,Cannot read the extension from
/c:/Users/consu/.vscode/extensions/redhat.java-1.24.0-win32-x64,Cannot read
the extension from /c:/Users/consu/.vscode/extensions/vscjava.
vscode-java-debug-0.55.0,Cannot read the extension from
/c:/Users/consu/.vscode/extensions/vscjava.vscode-maven-0.43.0,Cannot read
the extension from /c:/Users/consu/.vscode/extensions/vscjava.
vscode-java-dependency-0.23.2
    at c:\Users\consu\AppData\Local\Programs\Microsoft VS
Code\resources\app\out\vs\code\node\sharedProcess\sharedProcessMain.js:99:78806
    at Array.reduce (<anonymous>)
    at E (c:\Users\consu\AppData\Local\Programs\Microsoft VS
Code\resources\app\out\vs\code\node\sharedProcess\sharedProcessMain.js:99:78792)
    at K.M (c:\Users\consu\AppData\Local\Programs\Microsoft VS
Code\resources\app\out\vs\code\node\sharedProcess\sharedProcessMain.js:99:70467)
    at async K.J (c:\Users\consu\AppData\Local\Programs\Microsoft VS
Code\resources\app\out\vs\code\node\sharedProcess\sharedProcessMain.js:99:68153)
    at async c:\Users\consu\AppData\Local\Programs\Microsoft VS
Code\resources\app\out\vs\code\node\sharedProcess\sharedProcessMain.js:99:64821
    at async Promise.allSettled (index 0)
    at async K.I (c:\Users\consu\AppData\Local\Programs\Microsoft VS
Code\resources\app\out\vs\code\node\sharedProcess\sharedProcessMain.js:99:64768)
    at async K.installGalleryExtensions
(c:\Users\consu\AppData\Local\Programs\Microsoft VS
Code\resources\app\out\vs\code\node\sharedProcess\sharedProcessMain.js:99:63217)
    at async K.installFromGallery
(c:\Users\consu\AppData\Local\Programs\Microsoft VS
Code\resources\app\out\vs\code\node\sharedProcess\sharedProcessMain.js:99:62538)
